# This message consists of anything that needs updating
# for each sensor
uint16[] sensor_responses
int64[] sensor_receive_times
